Cubical/Algebra/CONTRIBUTING.md

```diff
@@ -0,0 +1,91 @@
+CONTRIBUTING
+============
+
+# General
+
+- We divide the definition of an algebraic structure into four parts:
+
+  1. Carrier: This is often `Type`, but might be for example `Type × Type`.
+  2. Properties of the carrier: this is usually `isSet`.
+  3. Operations: Non-propositional operations on the carrier, such as the multiplication and unit for a monoid.
+  4. Properties of the operations: for example, the associativity and unit laws for a monoid.
+
+  We separate (ii) and (iv) because it makes it easier to avoid redundant assumptions of `isSet` in complex
```

Comment:
maybe refer to them as "2." and "4."?

Reply:
Markdown renders the list items as i-iv and I don't know if one can change this. if only I could \ref...